pyintegration/tests/test_table.py

```diff
@@ -283,7 +284,6 @@ def test_agg_by_2(self):
                 weighted_avg("var", ["weights"])]
 
         result_table = test_table.agg_by(aggs, ["dumb"])
```

Comment:
Not part of this review, but I think the variable name `dumb` is "improper". I'd advise choosing completely "neutral" names commonly used (`foo`, `bar`, `baz`), or choose something "meaningful" such as `key`, or `my_group`. I'd suggest an audit of the client for other potentially "improper" names.

Reply:
will do.